griptape/executors/local_executor.py

```diff
@@ -15,18 +15,22 @@
 class LocalExecutor(BaseExecutor):
     verbose: int = field(default=False, kw_only=True)
 
-    def try_execute(self, tool_activity: callable, value: Optional[dict]) -> Union[BaseArtifact, str]:
+    dependencies_install_directory = field(default=None, kw_only=True)
```

Comment:
@vasinov should this be moved up to `BaseExecutor` too? And maybe with some tweaks this could accomplish what you had in mind for https://github.com/griptape-ai/griptape/issues/54?